mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/plugins/Authentication.kt

```diff
@@ -103,8 +102,8 @@ fun <T : Any> PipelineContext<T, ApplicationCall>.getNavAnsattAzureId(): UUID {
 }
 
 fun <T : Any> PipelineContext<T, ApplicationCall>.getNorskIdent(): String {
-    return call.request.header("nav-norskident") ?: throw StatusException(
+    return call.request.queryParameters["fnr"] ?: throw StatusException(
```

Comment:
Kanskje vi kan slenge på en enkel validering her også? min length 11 burde vel funke for norsk ident?

Reply:
Godt poeng, det bør vi ha ja.

Reply:
Har ikke helt oversikt over hvor mange varianter det finnes av norsk ident, kjenner bare til dnr og fnr, men har hørt at det finnes en annen variant også